robosherlock/src/core/include/robosherlock/scene_cas.h

```diff
@@ -273,6 +273,26 @@ class SceneCas
     }
     setFS(name, array);
   }
+
+public:
+  /**
+   * Convert the CAS to std::string, optional all the ByteArray and Integer Array are removed (point clouds and depth)
+   * @param CAS that should be converted
+   * @param saveWithPointCloud, if TRUE then the whole point clouds are converted too, if FALSE point clouds are removed
+   * @return std::string that contains the cas
+   */
+  std::string getCAStoString(uima::CAS &tcas, bool saveWithPointCloud);
```

Comment:
A minor thing regarding semantics:
Could you please rename the method to getCASAsString ?

And the string is containing a XML, right?
If that's correct, please add a remark into the return description of the method documentation. Thanks!

Reply:
changed the name and a bit of the description in the newest commit the string is just the cas literally as a string with either the point clouds in or not, only the saveCas writes the string as xml on to the machine